### PR TITLE
Refactor liquidity page list

### LINKED ISSUE #888: Liquidity page - connected

### PR DESCRIPTION
**Pull request by berteotti**
### WIP 
Me and @Diogomartf are working on updating the design of liquidity page and also the fonts and colors.

This is **work in progress** it's not finished, but we want to divide the work in smaller chunks and do things incrementally. 

We will do this smaller PRs and merge it on a "main" branch, `feature/fonts-and-colors-liquidity-page`.

--- 

# Summary

Closes #888

Refactor liquidity page list

Screenshot 2022-05-02 at 14 46 26

### CODE REVIEW

**Comment by MilanVojnovic95** [comment]
Hey @berteotti 
I am currently writing and almost done an automation test for Add Liquidity in which I interact with this page, if possible it is important to me that it remain the `data-testid` I wrote in the `index.tsx` because I use them for the selectors in test.

**Comment by berteotti** [comment]
> Hey @berteotti I am currently writing and almost done an automation test for Add Liquidity in which I interact with this page, if possible it is important to me that it remain the `data-testid` I wrote in the `index.tsx` because I use them for the selectors in test.

Thanks for the remainder! Will do it.

File: src/components/Pool/LoadingList/LoadingCard/index.tsx [hunk 1]
```diff
@@ -1,140 +1,40 @@
+// @ts-nocheck
```
Comment (Diogomartf) [inline src/components/Pool/LoadingList/LoadingCard/index.tsx:1]: I think we don't want to leave typecript disabled

**Review by Diogomartf — APPROVED** [review]

File: src/components/Pool/LoadingList/LoadingCard/index.tsx (hunk 1 above)
Comment (berteotti) [reply src/components/Pool/LoadingList/LoadingCard/index.tsx:1]: Thanks, fixed

**Pull request merged**